tests/test_enterprise/views/test_grant_data_sharing_permissions.py

```diff
@@ -372,17 +383,19 @@ def test_get_course_specific_consent_not_needed(
         assert response.status_code == 404
 
     @mock.patch('enterprise.views.render', side_effect=fake_render)
+    @mock.patch('enterprise.views.EnrollmentApiClient')
     @mock.patch('enterprise.api_client.discovery.CourseCatalogApiServiceClient')
     @mock.patch('enterprise.views.reverse')
     @ddt.data(
-        (True, True, '/successful_enrollment', 'course-v1:edX+DemoX+Demo_Course'),
-        (False, True, '/successful_enrollment', 'course-v1:edX+DemoX+Demo_Course'),
-        (True, False, '/failure_url', 'course-v1:edX+DemoX+Demo_Course'),
-        (False, False, '/failure_url', 'course-v1:edX+DemoX+Demo_Course'),
-        (True, True, '/successful_enrollment', 'edX+DemoX'),
-        (False, True, '/successful_enrollment', 'edX+DemoX'),
-        (True, False, '/failure_url', 'edX+DemoX'),
-        (False, False, '/failure_url', 'edX+DemoX'),
+        (True, True, '/successful_enrollment', 'course-v1:edX+DemoX+Demo_Course', ''),
+        (False, True, '/successful_enrollment', 'course-v1:edX+DemoX+Demo_Course', str(uuid.uuid4())),
+        (False, True, '/successful_enrollment', 'course-v1:edX+DemoX+Demo_Course', ''),
+        (True, False, '/failure_url', 'course-v1:edX+DemoX+Demo_Course', ''),
+        (False, False, '/failure_url', 'course-v1:edX+DemoX+Demo_Course', ''),
+        (True, True, '/successful_enrollment', 'edX+DemoX', ''),
+        (False, True, '/successful_enrollment', 'edX+DemoX', ''),
+        (True, False, '/failure_url', 'edX+DemoX', ''),
+        (False, False, '/failure_url', 'edX+DemoX', ''),
```

Comment:
Total nit that you can ignore here but might find useful in the future - this style:
```python
{
    'defer_creation': True, 'consent_provided': False, 'expected_redirect_url': '/successful_enrollment', 
    'course_id': 'course-v1:edX+DemoX+Demo_Course', 'license_uuid': ''
},
...
```
...while more verbose, does make it easier to understand at a glance what each set of data is actually testing.  And it works with `ddt.unpack` the same as tuples, so you won't have to change the signature of this test function.

Reply:
Agreed, since these tests existed before this PR I'm going to leave as is but I'll definitely be sure to use this syntax in the future.